migration/2.1.3/SequencerInbox.sol

```diff
@@ -57,779 +30,859 @@ import {IEspressoTEEVerifier} from "../bridge/IEspressoTEEVerifier.sol";
+      IBridge.BatchDataLocation.TxInput
+    );
+  }
+
+  function addSequencerL2BatchFromOrigin(
+    uint256 sequenceNumber,
+    bytes calldata data,
+    uint256 afterDelayedMessagesRead,
+    IGasRefunder gasRefunder,
+    uint256 prevMessageCount,
+    uint256 newMessageCount,
+    bytes memory quote
+  ) external refundsGas(gasRefunder, IReader4844(address(0))) {
+    revert Deprecated();
+  }
+
+  function addSequencerL2BatchFromBlobs(
+    uint256 sequenceNumber,
+    uint256 afterDelayedMessagesRead,
+    IGasRefunder gasRefunder,
+    uint256 prevMessageCount,
+    uint256 newMessageCount
+  ) external refundsGas(gasRefunder, reader4844) {
+    if (!isBatchPoster[msg.sender]) revert NotBatchPoster();
+
+    (
+      bytes32 dataHash,
+      IBridge.TimeBounds memory timeBounds,
+      uint256 blobGas
+    ) = formBlobDataHash(afterDelayedMessagesRead);
+
+    // we use addSequencerL2BatchImpl for submitting the message
+    // normally this would also submit a batch spending report but that is skipped if we pass
+    // an empty call data size, then we submit a separate batch spending report later
+    (
+      uint256 seqMessageIndex,
+      bytes32 beforeAcc,
+      bytes32 delayedAcc,
+      bytes32 afterAcc
+    ) = addSequencerL2BatchImpl(
+        dataHash,
+        afterDelayedMessagesRead,
+        0,
+        prevMessageCount,
+        newMessageCount
+      );
+
+    uint256 _sequenceNumber = sequenceNumber; // stack workaround
+
+    // ~uint256(0) is type(uint256).max, but ever so slightly cheaper
+    if (seqMessageIndex != _sequenceNumber && _sequenceNumber != ~uint256(0)) {
+      revert BadSequencerNumber(seqMessageIndex, _sequenceNumber);
+    }
+
+    emit SequencerBatchDelivered(
+      _sequenceNumber,
+      beforeAcc,
+      afterAcc,
+      delayedAcc,
+      totalDelayedMessagesRead,
+      timeBounds,
+      IBridge.BatchDataLocation.Blob
+    );
+
+    // blobs are currently not supported on host arbitrum chains, when support is added it may
+    // consume gas in a different way to L1, so explicitly block host arb chains so that if support for blobs
+    // on arb is added it will need to explicitly turned on in the sequencer inbox
+    if (hostChainIsArbitrum) revert DataBlobsNotSupported();
+
+    // submit a batch spending report to refund the entity that produced the blob batch data
+    // same as using calldata, we only submit spending report if the caller is the origin and is codeless
+    // such that one cannot "double-claim" batch posting refund in the same tx
+    if (CallerChecker.isCallerCodelessOrigin() && !isUsingFeeToken) {
+      submitBatchSpendingReport(
+        dataHash,
+        seqMessageIndex,
+        block.basefee,
+        blobGas
+      );
+    }
+  }
+
+  function addSequencerL2BatchFromBlobs(
+    uint256 sequenceNumber,
+    uint256 afterDelayedMessagesRead,
+    IGasRefunder gasRefunder,
+    uint256 prevMessageCount,
+    uint256 newMessageCount,
+    bytes memory quote
```

Comment:
Why is the only `addSequencerL2BatchFromBlobs` function with the `quote` parameter `deprecated`?